src/core.py

```diff
@@ -52,6 +56,10 @@ def on_config(self, config):
         )
         config["theme"].dirs.append(self.tmp_dir_techdocs_theme.name)
 
+        for key in theme_override:
+            if key not in config["theme"]:
+                config[key] = theme_override[key]
```

Comment:
I wonder if we should do this with a little more consideration for the values set above.  We, for example, don't want to allow the theme itself to be overridden entirely (e.g. the name), and we'd likely want to merge in the `static_templates` and `dirs` in, rather than overwriting them entirely.

Whaddya think?

Reply:
Thx for your feedback,

I think I've a mistake in my code, becose normally I want to protect replacement of existing key, for example:

1/ If "name" is declared in theme_override
2/ but already exist in config["theme"] 
3/ "name" is not overriden.

Reply:
@iamEAP I have updated my PR to follow your recommendation. 

for the moment it's ok from my side, do you have another feedback or feature to improve that, or is it ok for the first step? :)

Reply:
Thanks @aladjadj!  _Seems_ good to me, but I am admittedly not very familiar with python.

Perhaps @emmaindal, who is on community rotation this week, can chime in sometime today?